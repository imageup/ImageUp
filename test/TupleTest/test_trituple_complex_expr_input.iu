function main() -> int
{
    int : x = 1;
    int : y = 2;
    float : a = 3.0;
    float : b = 4.0;
    tuple:tu = (x*y-2, a+b*2.0, 0.5*b+3.0);
    printf(tu[0]);
    printf(tu[1]);
    printf(tu[2]);
    prints("trituple output should be: 0, 11, 5");
    return 0;
}