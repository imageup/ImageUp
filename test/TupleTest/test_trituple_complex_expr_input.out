0
11
5
trituple output should be: 0, 11, 5
